Sprint1/main.py

```diff
@@ -0,0 +1,150 @@
+from flask import Flask, render_template, request, redirect, url_for, session
+from flask_mysqldb import MySQL
+import MySQLdb.cursors
+import re
+
+app = Flask(__name__)
+
+
+app.secret_key = 'your secret key'
+
+# Enter your database connection details below
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_USER'] = 'mussaed'
+app.config['MYSQL_PASSWORD'] = 'mussaed0566060299'

```

Comment:
@npvergara @DerrekWoodworth We are gonna have to start thinking about environmentalizing these db credentials and other secrets so that they are not hard coded nor environment specific